style.css

```diff
@@ -13,40 +19,31 @@ body {
     height: 60px;
     font-size: 24px;
     margin: 10px;
-
     display: none;
     justify-content: space-between;
     padding: 0 10px;
     align-items: center;
 }
 
+.start {
+    background-color: red;
+    color : yellow;
+    width: 100px;
+    margin : 0 auto;
+    z-index: 9999;
```

Comment:
Convention way is people have other reasons why they want to do it that way. We don't have any such reasons.